freeipa/src/main/java/com/sequenceiq/freeipa/converter/backup/BackupConverter.java

```diff
@@ -62,4 +75,28 @@ private void decorateBackupFromLoggingRequest(Backup backup, LoggingRequest logg
             }
         }
     }
+
+    private void decorateBackupFromBackupRequest(Backup backup, BackupRequest backupRequest) {
+        if (backupRequest != null) {
```

Comment:
I think you should check both backup and request against null or neither of them

Reply:
Thanks! I checked both now. Indeed it makes sense.